Makefile

```diff
@@ -56,13 +56,12 @@ gen: ${GENALL}
 build-self: ensure-docker
 	(cd libs; cargo build --all)
 	(cd libs/gl-client-py && python3 -m maturin develop)
+	pip install -e libs/gl-testing
```

Comment:
Super helpful! I always did this manually

Reply:
Yeah, the scripts aren't quite there yet, but if we can make ours and our user's life easier we should do it.